src/components/Upload.vue

```diff
@@ -1,17 +1,20 @@
 <template lang="pug">
-.upload-wrapper
-  slot(name="toolbar")
-    v-toolbar(dense, dark, color="success")
-      v-toolbar-title Upload
-      v-spacer
-      v-btn(v-if="closeable", icon, flat, @click="$emit('close')")
-        v-icon close
-  .px-0.py-2
-    slot(name="header")
-      .title.px-3
-        v-avatar.mr-2(color="blue-grey lighten-4")
-          v-icon {{ ResourceIcons[dest._modelType.toUpperCase()] }}
-        span {{ dest.name }}
+v-card.upload-wrapper
```

Comment:
I think we should do `v-card.upload-wrapper(flat)` so that it doesn't get a border-radius and box-shadow on the root element.

Reply:
+1 done.